### ISSUE TITLE
ES refresh interval not reset correctly after import

### ISSUE THREAD

**Issue by mooreds** [FusionAuth/fusionauth-issues#924]
### Description 

After an import of 3M users, it took tens of minutes to have the search index updated completely. 

Looked at the ES logs, it looks like the refresh interval didn’t get set back to to the correct interval after the imports.During an import request we tell ES not to refresh the index so that it doesn’t thrash while we are pumping data into it quickly.
At the end of each import we set it back to whatever value it was previously.

### Affects versions

1.19.8

### Steps to reproduce

1. Import a bunch of users
1. Wait for ES to catch up

### Expected behavior

It should catch up quicker.

### Additional context

From https://fusionauth.slack.com/archives/GNURTTC8N/p1602103931491200

**Comment by robotdan** [comment]
I think the cause of this is that we ask ES for the current refresh interval, and then we the import request is complete we set it back to whatever it was previously. 

In theory, if you were to try to run multiple import requests, we would ask ES for the current value which would be the disabled value that we set during the migration. (`-1`). 

We could just set it back to the ES default value at the end, this setting is per index, so we won't clobber any other index settings on the ES instance, or we could be more defensive on resetting this interval refresh value at the end. For example, we could check if the current value is `-1` we know that is not the correct "original" value.
